.lvimrc

```diff
@@ -0,0 +1,3 @@
+set expandtab
```

Comment:
Huh, I've not used this plugin before.

What about adopting editorconfig instead?  It'd be applicable to more contributors.